utils/sample_qc.py

```diff
@@ -363,99 +363,114 @@ def infer_sex(
     return (sex_ht)
 
 
+def get_ploidy_cutoffs(
+    ht: hl.Table, 
+    f_stat_cutoff: float, 
+    normal_ploidy_cutoff: int = 5, 
+    aneuploidy_cutoff: int = 6
+) -> tuple:
```

Comment:
`tuple` won't work, we need the typing class `Tuple`. If you  agree with my  proposals, then the full return type would be:
`Tuple[Tuple[float, Tuple[float, float], float], Tuple[Tuple[float, float], float]]`